pynucastro/networks/tests/_starkiller_cxx_reference/actual_rhs.H

```diff
@@ -193,13 +193,11 @@ void rhs_nuc(const burn_t& state,
         0.5*screened_rates(k_c12_c12__he4_ne20)*std::pow(Y(C12), 2)*state.rho;
 
     ydot_nuc(Ne23) =
-        -screened_rates(k_ne23__na23)*Y(Ne23) +
-        screened_rates(k_na23__ne23)*Y(Na23);
+        (-screened_rates(k_ne23__na23)*Y(Ne23) + screened_rates(k_na23__ne23)*Y(Na23));
 
     ydot_nuc(Na23) =
-        -screened_rates(k_na23__ne23)*Y(Na23) +
         0.5*screened_rates(k_c12_c12__p_na23)*std::pow(Y(C12), 2)*state.rho +
-        screened_rates(k_ne23__na23)*Y(Ne23);
+        (screened_rates(k_ne23__na23)*Y(Ne23) + -screened_rates(k_na23__ne23)*Y(Na23));
```

Comment:
there is typo here +-

Reply:
that's not a typo.  The `-` is the "unary minus" and the `+` is the operator.  The compiler probably will optimize this out.

Reply:
ohh this is the fist time I see it. My pardon.